rand_chacha/Cargo.toml

```diff
@@ -16,9 +16,11 @@ edition = "2018"
 
 [dependencies]
 rand_core = { path = "../rand_core", version = "0.6.0" }
-ppv-lite86 = { version = "0.2.8", default-features = false, features = ["simd"] }
+ppv-lite86 = { version = "0.2.10", default-features = false, features = ["simd"] }
+serde = { version = "1.0", features = ["derive"], optional = true }
 
 [features]
-default = ["std"]
+default = ["std", "serde1"]
```

Comment:
Please don't make it a default feature. Serde takes significant time to compile, and not everyone is using it.

Reply:
Oops I just had that in for a quick test.